nss-day-cohort-60/national-parks-api
# Md sql helper functions for update and delete

Description:

1. Added new sql helper functions called update_resource and delete_resource to sql_helper.py 
2. Implemented these functions into wildlife_views.py and natural_attractions.py

Steps to test:
1. Fetch the md-PUT-sql-helper-function branch and switch to it
2. restart debugger
3. Open Postman and make the following requests:

- Send a PUT request to update the selected wildlife(e.g. http://localhost:8088/wildlife/1)
- {
        "name": "A Big Cat",
        "information": "New info about this animal.",
        "wildlife_group_id": 1,
        "image": "https://res.cloudinary.com/dcuwovsbv/image/upload/v1674244197/Wildlife/Mountain-Lion-Puma-concolor.jpg_wbe7ck.webp"
    }
- Verify that a 204 status code was received and switch back to the GET function
- Request all wildlife, and verify that the dictionary was updated
- Send a DELETE request to delete the specified dictionary (e.g. http://localhost:8088/wildlife/1)
- Switch back to GET and request all wildlife to verify that it was removed

Repeat this process to test PUT and DELETE for natural_resources
